modules/standard/SysBasic.chpl

```diff
@@ -41,7 +42,7 @@ deprecated "'qio_err_t' has been deprecated; please use a 'CTypes.c_int' instead
 extern type qio_err_t = c_int;
 
 /* A system file descriptor. This is really just a `c_int`, but code is
-   clearer if you use fd_t to indicate arguments, variables, and return types
+   clearer if you use c_int to indicate arguments, variables, and return types
```

Comment:
This comment update is a little redundant, so I'd maybe just leave it alone since we're deprecating the module as a whole

Reply:
Whoops -- find and replace